src/config.ts

```diff
@@ -343,6 +332,13 @@ export class MirrordConfigManager {
         return null;
       }
     } else {
+      // User probably openend vscode in a single file, no folder is loaded and they have
+      // not set up the `MIRRORD_CONFIG_FILE` env var.
+      new NotificationBuilder()
+        .withMessage(`Could not find a config file, using the default mirrord configuration.`)
+        .withDisableAction("promptUsingDefaultConfig")
```

Comment:
In the other case where we use that action it's where we use a configuration file in the default location, here we don't use any config file. So I would say it should be a different action.

Reply:
Changed.